<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ALMainWindow</class>
 <widget class="QMainWindow" name="ALMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AutoLibrary</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <layout class="QVBoxLayout" name="CentralWidgetLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="ControlLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="TimerTaskWidgetButton">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="document-open-recent"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="ControlSpaceFrame">
        <property name="minimumSize">
         <size>
          <width>1000</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ConfigButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>配置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="StopButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>停止脚本</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="StartButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;开始运行自动脚本&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(10, 170, 10);
font: 12pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);
font: 9pt &quot;Segoe UI&quot;;
font: 700 9pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>启动脚本</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="MessageIOTextEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>175</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QMenu::icon {
        width: 0px;
        height: 0px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <property name="tabChangesFocus">
       <bool>false</bool>
      </property>
      <property name="undoRedoEnabled">
       <bool>false</bool>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::LineWrapMode::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextInteractionFlag::LinksAccessibleByKeyboard|Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextBrowserInteraction|Qt::TextInteractionFlag::TextEditable|Qt::TextInteractionFlag::TextEditorInteraction|Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
      <property name="centerOnScroll">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="MessageLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <item>
       <widget class="QLineEdit" name="MessageEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SendButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
        <property name="icon">
         <iconset theme="document-send"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>33</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="HelpMenu">
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="ManualAction"/>
    <addaction name="AboutAction"/>
   </widget>
   <addaction name="HelpMenu"/>
  </widget>
  <widget class="QStatusBar" name="StatusBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="ManualAction">
   <property name="text">
    <string>在线手册</string>
   </property>
  </action>
  <action name="AboutAction">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
